### ISSUE TITLE
Include NewsGuard extension by default

### ISSUE THREAD

**Issue by domcorvasce** [brave/brave-browser#5417]
## Description 

> NewsGuard uses journalism to fight false news, misinformation, and disinformation.
> Our trained analysts, who are experienced journalists, research online news brands to help readers and viewers know which ones are trying to do legitimate journalism—and which are not.
>
> [NewsGuard website](http://www.newsguardtech.com/)

I am in no way associated with them, but I think I'd be useful if Brave included this extension by default -- like Microsoft Edge does/did.

## Actual result:

The extension must be installed manually, via https://chrome.google.com/webstore/detail/newsguard/hcgajcpgaalgpeholhdooeddllhedegi?hl=en.

## Expected result:

The extension is already installed in Brave.

**Comment by kjozwiak** [comment]
I don't think we have plans to add certain extensions into Brave my default at this moment. Users always have a choice of download/installing any extension via the GPS. Adding certain extensions might benefit a small cohort of users, but the majority of the Brave users will most likely never benefit from it. CCing @rebron.

**Comment by domcorvasce** [comment]
I see your point, though I don’t fully agree about the fact that it would benefit only a cohort of users.

Although users can install it manually, there always is a certain friction.

However, thanks for the quick response and keep up the good work with Brave!

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
